spl/src/governance.rs

```diff
@@ -46,8 +49,15 @@ macro_rules! vote_weight_record {
             }
         }
 
+        impl anchor_lang::Discriminator for VoterWeightRecord {
```

Comment:
I think implementing Discriminator for spl-govs VoterWeightRecord is misleading, since contrary to anchor's usual accounts, it's not separate from the account struct's data.

Reply:
agreed